lib/engine.py

```diff
@@ -698,16 +696,29 @@ def table_name(self, name=None, dbname=None):
         return self.opts["table_name"].format(db=dbname, table=name)
 
     def warning(self, warning):
-        new_warning = Warning('%s:%s' % (self.script.shortname, self.table.name), warning)
+        new_warning = Warning('%s:%s' % (
+            self.script.shortname, self.table.name), warning)
         self.warnings.append(new_warning)
 
 
+def load_data(filename, delimiter="\t"):
```

Comment:
I don't understand what this function is doing. It would definitely benefit from a doc string describing what it is doing and why?